packages/react-percy-api-client/src/resources/__tests__/makeRootResource-tests.js

```diff
@@ -21,6 +21,19 @@ it('sets resource URL to slug-ified snapshot name', () => {
   );
 });
 
+it('strips quotes from snapshot name when setting resource URL', () => {
+  const snapshotName = 'Suite - renders "my' + "' component";
+  const html = '<html></html>';
+
+  const rootResource = makeRootResource(percyClient, snapshotName, html);
+
+  expect(rootResource).toEqual(
+    expect.objectContaining({
```

Comment:
Haven't seen this nested expect syntax before, but looks like it's a pattern in this framework.